src/components/select/select-textbox/select-textbox.spec.tsx

```diff
@@ -185,15 +159,15 @@ describe("SelectTextbox", () => {
     });
 
     describe.each([
-      ["set", "ariaLabelledBy", "foo"],
-      ["not set", "labelId", undefined],
+      ["set", "foo"],
+      ["not set", undefined],
     ])(
       "when the ariaLabelledBy prop is %s",
-      (description, propName, mockAriaLabelledBy) => {
+      (description, mockAriaLabelledBy) => {
         const mockLabelId = "baz";
 
         describe("with the hasTextCursor prop set to true", () => {
-          it(`then the aria-labelledby in the Textbox should contain ${propName} and the id of the SelectText`, () => {
+          it(`then the aria-labelledby in the Textbox should be the value of the ariaLabelledBy prop`, () => {
```

Comment:
**nitpick (non-blocking):** I know this `it` was a template literal/string for string interpolation, but as you've edited it, and it is now just a string, it might be worth changing it back to quotes, to make sure the use of backticks and quotations are consistent.

Reply:
Excellent point thanks, I'd missed this. I'll get it changed.